File: desafio-02/Viniciustertuliano/java/NumerosPrimos.java
public class NumerosPrimos {
    public static void main(String[] args) {
        for (int i = 2; i <= 10000; i++) {
            if(primo(i)){
                System.out.println(i);
            }
        }
    }

    private static boolean primo(int numero) {
        for (int a = 2; a < numero; a++) {
            if(numero % a == 0){
                return false;
            }
        }
        return true;
    }
}

File: desafio-02/devfelipelimabr/numerosPrimos.js
function primo(numero = 10000) {

    let primos = [2, 3, 5, 7];
    for (let i = 8; i <= numero; i++) {
        if (Number.isInteger(i % 2) === false && Number.isInteger(i % 3) === false && Number.isInteger(i % 5) === false && Number.isInteger(i % 7) === false) {
            primos.push(i);
        }
    }
    return primos;
}

let j = 11;
console.log(Number.isInteger(j % 2));